File: easy/countingBits/dart/countingBits.dart
class Solution {
  List<int> countBits(int n) {
      List<int> ans = [];
      for (var i = 0; i <= n ; i++) {
          ans.add('1'.allMatches(i.toRadixString(2)).length);
      }
      return ans;
  }
}

File: easy/countingBits/php/countingBits.php
<?php

class Solution {

/**
 * @param Integer $n
 * @return Integer[]
 */
    function countBits($n): array {
        $ans = [];
        for($i = 0; $i <= $n; $i++) {
            $ans[$i] = substr_count(decbin($i),'1');
        }
        return $ans;
    }
}